modules/EvseV2G/connection.cpp

```diff
@@ -644,8 +664,12 @@ static void ssl_key_log_debug_callback(void* ACtx, int ALevel, const char* AFile
     ctx->hexdumpLinesToProcess = 0;
     ctx->inClientRandom = false;
     ctx->inMasterSecret = false;
-    fputc('\n', ctx->file); /* finish key log line */
+    const auto message = '\n';
+    fputc(message, ctx->file); /* finish key log line */
+    add_character_to_udp_message(message);
     fflush(ctx->file);
+    send_udp_message();
+    ctx->udp_buffer = {};
```

Comment:
I would not intertwine the udp and FILE handling like this, better do them one after another